definitions/npm/react-with-separator_v1.x.x/flow_v0.72.x-/test_react-with-separator_v1.x.x.js

```diff
@@ -0,0 +1,29 @@
+import React from "react";
+import WithSeparator from "react-with-separator";
```

Comment:
Please use `it/describe` blocks for better organization.

Reply:
Ah. You're correct in that the test runner doesn't actually use them. We use `describe/it` from the test helper file because it just makes tests a bit more expressive. See [here](https://github.com/flow-typed/flow-typed/pull/2945/files#diff-f9434d4cd5132206735a09150d0876f8) for an example usage.

Reply:
:white_check_mark: